src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,13 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    private Random random = new Random();
+
     public String getRandomColor() {
-        return null;
+        int index = random.nextInt(Color.values().length);
+        Color color = Color.values()[index];
+        return color.name();
     }
```

Comment:
I believe we do not really need `color` variable here. Feel free to return its' `name()` representation just from the line 10.